helm/batch-processing-gateway/values.yaml

```diff
@@ -22,7 +22,128 @@ ingress:
 
 # Config is provided as a base64-encoded,
 # compressed content which will be projected into pods as /app/app-config.gz.
-encodedConfig: ''
+encodedConfig:
+
+plainConfig:
+  defaultSparkConf:
+    spark.kubernetes.submission.connectionTimeout: 30000
+    spark.kubernetes.submission.requestTimeout: 30000
+    spark.kubernetes.driver.connectionTimeout: 30000
+    spark.kubernetes.driver.requestTimeout: 30000
+    spark.sql.debug.maxToStringFields: 75
+  sparkClusters:
+    - weight: 100
+      id: c0101
+      eksCluster: cluster-1
+      masterUrl: https://34.134.183.114
```

Comment:
is this a dummy value? can you make it an empty string or make it more obvious that it's a dummy value?

Reply:
yes, should be a dummy value